array/moon.pkg.json

```diff
@@ -1,12 +1,12 @@
 {
   "import": [
     "moonbitlang/core/builtin",
-
+    "moonbitlang/core/bytes",
```

Comment:
This extra dep is used because `blit_from_bytesview` is moved from `@builtin` to `@array`.